package.json

```diff
@@ -22,7 +22,7 @@
     "url": "http://github.com/home-assistant/homebridge-homeassistant/issues"
   },
   "engines": {
-    "node": ">=0.12.0",
+    "node": ">=6.0.0",
```

Comment:
I agree that I don't need to bump the version to use the feature (class keyword) that I need to achieve the goal (inheritance). @robbiet480 said this would be OK, which is why I went ahead with it.